index.php

```diff
@@ -24,15 +20,15 @@
   </header>
   <section id="guitars">
       <?php
-      foreach ($data as $guitar) { ?>
+      foreach ($guitars as $guitar) { ?>
```

Comment:
thought you put this foreach into a function? this is fine but id rather this foreach be inside a function